src/PhysicsEngine.cpp

```diff
@@ -484,3 +494,65 @@ Quaterniond PhysicsEngine::update_quaternion(Quaterniond q_ornt,
 
     return q_ornt;
 }
+
+/**
+ * @brief 
+ * 
+ * @param pos_if 
+ * @return Vector3d 
+ */
+Vector3d RungeKutta::i2ecef(Vector3d pos_if) {
```

Comment:
the unit conversion functions should be part of rocket imo. Since they depend on the launch pad information that is stored in rocket.